app/models/course.rb

```diff
@@ -267,6 +269,17 @@ def notify_accredited_body?
     (saved_changes.keys & update_notification_attributes).any?
   end
 
+  def send_create_notification_to_accredited_body
+    users = UserNotification.where(provider_code: accrediting_provider_code, create_course: true).map(&:user)
```

Comment:
We should make this a scope on `UserNotification` rather than coupling to the DB fields from here. e.g. `UserNotification.users_requesting_create_course_notification_for(provider: <code>)` or sumfink.